File: CodingTest/Python/solution2.py
# https://school.programmers.co.kr/learn/courses/30/lessons/42842
def solution(brown, yellow):
    answer = []
    total = brown + yellow
    for i in range(2, total//2):
        if total % i == 0:
            j = total // i
            if (i - 2) * (j - 2) == yellow:
                answer.append(max(i,j))
                answer.append(min(i,j))
                break
        
    return answer

File: CodingTest/Java/Solution2.java
// https://school.programmers.co.kr/learn/courses/30/lessons/12951
package CodingTest.Java;

public class Solution2 {
    public String solution(String s) {
        String answer = "";
        String[] temp = s.split("");
        boolean check = true;
        for (int i = 0; i < temp.length; i++) {
            if (check) {
                if (temp[i].equals(" ")) {
                    
                } else {
                    temp[i] = temp[i].toUpperCase();
                    check = false;
                }
            } else {
                if (temp[i].equals(" ")) {
                    check = true;
                } else {
                    temp[i] = temp[i].toLowerCase();
                }
            }
            answer += temp[i];
        }
        return answer;
    }
}
